packages/mask/src/plugins/Avatar/SNSAdaptor/index.tsx

```diff
@@ -1,9 +1,46 @@
 import type { Plugin } from '@masknet/plugin-infra'
+import { useAsync } from 'react-use'
+import { ApplicationEntry } from '@masknet/shared'
+import { Trans } from 'react-i18next'
+import { useRemoteControlledDialog } from '@masknet/shared-base-ui'
+import { WalletMessages } from '@masknet/plugin-wallet'
 import { base } from '../base'
+import { CollectiblesIcon } from '@masknet/icons'
+import { activatedSocialNetworkUI } from '../../../social-network'
+import { loadSocialNetworkUI } from '../../../social-network/ui'
 
 const sns: Plugin.SNSAdaptor.Definition = {
     ...base,
     init(signal) {},
+    ApplicationEntries: [
+        {
+            RenderEntryComponent({ disabled, icon, title }) {
+                const { value: ui } = useAsync(async () => {
+                    return loadSocialNetworkUI(activatedSocialNetworkUI.networkIdentifier)
```

Comment:
can you explain this?

Reply:
Each SNS defines its function to open NFT avatar setting dialog respectively.